charts/mirrorbits-parent/templates/ingress.yaml

```diff
@@ -24,35 +24,18 @@ spec:
   {{- end }}
 {{- end }}
   rules:
-  {{- range .Values.ingress.hosts }}
+  {{- range .Values.global.ingress.hosts }}
     - host: {{ .host | quote }}
       http:
         paths:
         {{- range .paths }}
           - path: {{ .path }}
             pathType: {{ .pathType | default "Prefix" }}
             backend:
-            {{- if eq .backendService "mirrorbits" }}
-              {{- if not (index $.Values "mirrorbits-lite" "enabled") }}
-                {{- fail "Cannot use mirrorbits(-lite) as backend if it is disabled." }}
-              {{- end }}
               service:
-                # TODO: update child chart to use a global value to let the parent-chart defined/get the value
-                name: {{ $.Release.Name | trunc 63 | trimSuffix "-" -}}-mirrorbits-lite
+                name: {{ include "mirrorbits-parent.ingressBackendName" (dict "currentBackendService" .backendService "rootContext" $) }}
```

Comment:
I found some explanations about the dict passed as arg at https://v2.helm.sh/docs/charts_tips_and_tricks/, so "it's in the official doc", but a little comment wouldn't hurt there as it's an unusual syntax, WDYT?

Reply:
To be honest, I don't find it unusual at all and it is a key language feature (you found the doc quickly :) ).

Beside, the template function is documented about the expectation (e.g. the argument must be a map/dict). I realize that we might want to `s/map/dict` in the helper though WDYT?

Reply:
> we might want to s/map/dict in the helper though WDYT?

I agree!

Reply:
https://github.com/jenkins-infra/helm-charts/pull/813/commits/2627bc288d3eaa1e583d7d01bde5af594f120804